query_execution/QueryExecutionMessages.proto

```diff
@@ -0,0 +1,33 @@
+//   Copyright 2015-2016 Pivotal Software, Inc.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto2";
+
+package quickstep.serialization;
+
+// Used for both Normal WorkOrders and RebuildWorkOrders.
+message WorkOrderCompletionMessage {
+  required uint64 operator_index = 1;
+  required uint64 worker_id = 2;
+}
+
+message DataPipelineMessage {
+  required uint64 operator_index = 1;
+  required fixed64 block_id = 2;
+  required int32 relation_id = 3;
+}
+
+message WorkOrdersAvailableMessage {
+  required uint64 operator_index = 1;
+}
```

Comment:
I think these message classes are neat and help us to get rid of the C++ classes for the message types, which were very lightweight anyway. Thanks!